app/views/bus_stops/by_route.haml

```diff
@@ -3,14 +3,10 @@
 - if @stops.empty?
   No stops on this route
 - else
-  - @stops.each_pair do |complete, stops|
-    - if complete
-      Completed
-    - else
-      Not Completed
-    Stops
+  - @stops_hash.each_pair do |status, stops|
+    = status + ' Stops'
     %ul
     - stops.each do |stop|
       %li
         = link_to edit_bus_stop_path(stop.hastus_id) do
-          - stop.name
+          - stop.name
```

Comment:
Could you fix your text editor so it doesn't remove newlines from the end of files? I'm so sick of seeing this everywhere.

Reply:
I am a fan of reviewing with `?w=1` as the querystring.

Reply:
:+1: Or, there's actually an option in the "Diff settings" menu now.

<sub>Or, you know http://www.dealwithit.org/</sub>